manifest_base.yml

```diff
@@ -18,8 +18,6 @@ applications:
   health-check-type: process
   command: celery beat --app webservices.tasks
 - name: celery-worker
-  instances: 1
-  memory: 512M
```

Comment:
Do the apps default to these values when pushed to cloud.gov if they're not specified?